tool/pr_screenshot_publish_test.py

```
@@ -0,0 +1,143 @@
+from pathlib import Path
+
+from pr_screenshot_publish import (
+    CACHE_CONTROL,
+    PublishError,
+    discover_uploads,
+    publish_screenshots,
+)
+
+
+class _MissingObjectError(Exception):
+    response = {"Error": {"Code": "404"}}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)